week-2/L-array-methods-2/exercise2.js

```diff
@@ -7,7 +7,12 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  return str
+    .split()
+    .join()
+    .toUpperCase();
+}
```

Comment:
Your function change the input to uppercase but the expectation only changes the first letter to be capitalized